### PR TITLE
Add image-resource alias to image command

### PR DESCRIPTION
**Pull request by dumez-k**

### CODE REVIEW

**Comment by dumez-k** [comment]
Addresses: https://github.com/pivotal/kpack/issues/819

Waiting for PR into the kpack-cli repo before merging. 

There also seems to be some discussion left as to whether we want `image-resource` as a primary alias or as a secondary alias.

**Review by tylerphelan — APPROVED** [review]

**Comment by matthewmcnew** [comment]
I think we also want to audit any ["image configuration"](https://github.com/vmware-tanzu/kpack-cli/blob/928ce80e2cdeecf35d5ae594e214b05e6956b254/pkg/commands/image/create.go#L30) references and make sure we are consistent with image resource naming similar to proposed help text on https://github.com/pivotal/kpack/issues/819.

File: pkg/commands/build/status_test.go [hunk 1]
```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/sclevine/spec"
 	"github.com/spf13/cobra"
 	"github.com/stretchr/testify/assert"
+	"github.com/vmware-tanzu/kpack-cli/pkg/commands"
```
Comment (tylerphelan) [inline pkg/commands/build/status_test.go:18]: wow messed up our imports

**Review by tylerphelan — APPROVED** [review]
Nice! this is not easy to do!

**Review by tomkennedy513 — APPROVED** [review]

File: pkg/commands/build/status_test.go (hunk 1 above)
Comment (dumez-k) [reply pkg/commands/build/status_test.go]: fixed smh

File: pkg/commands/build/status.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func displayBuildStatus(cmd *cobra.Command, bld v1alpha2.Build) error {
 	}
 
 	statusItems := []string{
-		"Image", bld.Status.LatestImage,
+		"Image Resource", bld.Status.LatestImage,
```
Comment (matthewmcnew) [inline pkg/commands/build/status.go:128]: This is explicitly not the image resource it is the image itself.

File: pkg/commands/build/status.go [hunk 3]
```diff
@@ -40,14 +40,14 @@ func NewStatusCommand(clientSetProvider k8s.ClientSetProvider, rup registry.Util
...
-		Long: `Prints detailed information about the status of a specific build of an image in the provided namespace.
+		Short: "Display status for an image resource build",
+		Long: `Prints detailed information about the status of a specific build of an image resource in the provided namespace.
 
 The build defaults to the latest build number.
 The namespace defaults to the kubernetes current-context namespace.
 
 When using the --bom flag, only the built image's bill of materials will be printed.
-Using the --bom flag will read metadata from the build's built image in the registry
+Using the --bom flag will read metadata from the build's built image resource in the registry
```
Comment (matthewmcnew) [inline pkg/commands/build/status.go:50]: It is actually reading the image not the image resource in the registry.

File: pkg/commands/image/create.go [hunk 4]
```diff
@@ -27,9 +27,9 @@ func NewCreateCommand(clientSetProvider k8s.ClientSetProvider, rup registry.Util
 
 	cmd := &cobra.Command{
 		Use:   "create <name> --tag <tag>",
-		Short: "Create an image configuration",
-		Long: `Create an image configuration by providing command line arguments.
-This image will be created only if it does not exist in the provided namespace.
+		Short: "Create an image resource configuration",
```
Comment (matthewmcnew) [inline pkg/commands/image/create.go:30]:
```suggestion
		Short: "Create an image resource",
```

File: pkg/commands/image/create.go [hunk 5]
```diff
@@ -27,9 +27,9 @@ func NewCreateCommand(clientSetProvider k8s.ClientSetProvider, rup registry.Util
...
+		Long: `Create an image resource configuration by providing command line arguments.
```
Comment (matthewmcnew) [inline pkg/commands/image/create.go:31]:
```suggestion
		Long: `Create an image resource by providing command line arguments.
```

File: pkg/commands/image/patch.go [hunk 6]
```diff
@@ -28,9 +28,9 @@ func NewPatchCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilP
 
 	cmd := &cobra.Command{
 		Use:   "patch <name>",
-		Short: "Patch an existing image configuration",
-		Long: `Patch an existing image configuration by providing command line arguments.
-This will fail if the image does not exist in the provided namespace.
+		Short: "Patch an existing image resource configuration",
```
Comment (matthewmcnew) [inline pkg/commands/image/patch.go:31]:
```suggestion
		Short: "Patch an existing image resource",
```

File: pkg/commands/image/patch.go [hunk 7]
```diff
@@ -28,9 +28,9 @@ func NewPatchCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilP
...
+		Long: `Patch an existing image resource configuration by providing command line arguments.
```
Comment (matthewmcnew) [inline pkg/commands/image/patch.go:32]:
```suggestion
		Long: `Patch an existing image resource by providing command line arguments.
```

File: pkg/commands/image/save.go [hunk 8]
```diff
@@ -27,9 +27,9 @@ func NewSaveCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilPr
 
 	cmd := &cobra.Command{
 		Use:   "save <name> --tag <tag>",
-		Short: "Create or patch an image configuration",
-		Long: `Create or patch an image configuration by providing command line arguments.
-This image will be created only if it does not exist in the provided namespace, otherwise it will be patched.
+		Short: "Create or patch an image resource configuration",
```
Comment (matthewmcnew) [inline pkg/commands/image/save.go:30]:
```suggestion
		Short: "Create or patch an image resource",
```

Comment (matthewmcnew) [reply pkg/commands/image/save.go:30]: I think we should avoid "resource configuration" if we can.

**Review by matthewmcnew — CHANGES_REQUESTED** [review]
Could we avoid the phrasing resource configuration?

File: pkg/commands/build/status.go (hunk 2 above)
Comment (dumez-k) [reply pkg/commands/build/status.go:128]: > Could we avoid the phrasing resource configuration?

yes!

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**